controllers/storageoscluster/before_install_operand.go

```diff
@@ -126,10 +181,25 @@ func getAdditionalResources(ctx context.Context, c client.Client, obj client.Obj
 			return nil, err
 		}
 		if !crdExists {
-			resources = append(resources, crdFile)
+			crdFiles = append(crdFiles, crdFile)
 		}
 	}
-	return resources, nil
+
+	return crdFiles, nil
+}
+
+// getResourceQuota gets resource-quota file that need to be added as kustomize resources.
+func getResourceQuota(cluster *storageoscomv1.StorageOSCluster, kubeDistro string) (string, error) {
+	// Set the k8s distro if not already set.
+	if cluster.Spec.K8sDistro == "" {
+		cluster.Spec.K8sDistro = kubeDistro
+	}
+
+	if cluster.Spec.K8sDistro == distro.DistributionGKE {
```

Comment:
I would be happier to see something like this:
`resourceQuotaDistros := map[string]bool{distro.DistributionGKE: true}` and `_,ok := resourceQuotaDistros[cluster.Spec.K8sDistro]. It looks like a "declarative" solution instead of "imperative"

Reply:
nice 🥇